Sources/Basic/FSProxy.swift

```diff
@@ -268,19 +268,19 @@ private class LocalFS: FSProxy {
 //
 // FIXME: This class does not yet support concurrent mutation safely.
 public class PseudoFS: FSProxy {
-    private class Node {
+    fileprivate class Node {
```

Comment:
This surprises me, it is certainly the intent that this is `private` not `fileprivate`. What required this?